android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/notifications/NotificationsModule.java

```diff
@@ -96,7 +96,7 @@ public void deleteCategoryAsync(final String categoryIdParam, final Promise prom
   private String getScopedIdIfNotDetached(String categoryId) {
     if (!Constants.isStandaloneApp()) {
       try {
-        String experienceId = mManifest.getID();
+        String experienceId = mManifest.getStableLegacyID();
```

Comment:
Category-related things should use scope key (we are going to need to send down the scope key in notification payloads)